community/portworx/values.yaml

```
@@ -10,7 +10,7 @@ storage:
   usedrivesAndPartitions: false         # Defaults to false. Change to true and PX will use unmounted drives and partitions.
   drives: none                          # NOTE: This is a ";" seperated list of drives. For eg: "/dev/sda;/dev/sdb;/dev/sdc" Defaults to use -A switch.
   journalDevice: none
-  metadataSize: 64
+  metadataSize: none
```

Since this is a number, why is the default a string of `none` instead of `0`?
```suggestion
  metadataSize: 0
```